sale_financial_risk/models/res_partner.py

```diff
@@ -39,7 +39,7 @@ def _compute_risk_sale_order(self):
             domain=self._get_risk_sale_order_domain(),
             fields=["risk_partner_id", "company_id", "risk_amount"],
             groupby=["risk_partner_id", "company_id"],
-            orderby="id",
+            orderby="risk_partner_id, company_id",
```

Comment:
Why you need order by that fields?
For better performance use _read_group to avoid order.